maintainers/README.md

```diff
@@ -172,3 +172,49 @@ to the team without an approval by at least one existing member.
 Various utility scripts, which are mainly useful for nixpkgs maintainers,
 are available under `./scripts/`.  See its [README](./scripts/README.md)
 for further information.
+
+# Guidelines for Committers
+
+When merging pull requests, care must be taken to reduce impact to the `master`
+branch. If a commit breaks evaluation, it will affect Ofborg evaluation results
+in other pull requests and block Hydra CI, thus introducing chaos to our
+workflow.
+
+One approach to avoid merging such problematic changes is to wait for
+successful Ofborg evaluation. Additionally, using tools like `nixpkgs-review`
+can help spot issues early, before Ofborg finishes evaluation.
+
+## Breaking changes
+
+In general breaking changes to `master` and `staging` branches are permitted.
+Though restrictions might apply towards the end of a NixOS release cycle, due
+to our feature freeze mechanism. This is to avoid large-scale breakages shortly
+before and during a Zero Hydra Failures (ZHF) campaign. These restrictions also
+intend to decrease the likelihood of a delayed NixOS release. The feature
+freeze period is documented in the announcement of each release schedule.
+
+These are some example changes and if they are considered a breaking change
```

Comment:
This could be a nested paragraph, then it's clearer it's separate from the other guidelines and only counts for a limited time (around releases).